packages/protocol/contracts/L1/v1/V1Proving.sol

```diff
@@ -204,7 +204,7 @@ library V1Proving {
         address prover,
         bool enabled
     ) public {
-        assert(LibConstants.K_WHITELIST_PROVERS);
+        assert(s.whitelistProvers);
```

Comment:
Since we dont use constants anymore here, do we need to use `require` back and return a reason message? (doesn't really matter)

Reply:
"s.whitelistProvers" are not input of this function, so I'd prefer to use assert for states.